src/WorkOS.net/Services/Passwordless/_interfaces/CreatePasswordlessSessionOptions.cs

```diff
@@ -35,6 +35,15 @@ public class CreatePasswordlessSessionOptions : BaseOptions
         [JsonProperty("connection")]
         public string Connection { get; set; }
 
+        /// <summary>
+        /// An optional parameter. The number of seconds the Passwordless
+        /// Session should live before expiring.
+        /// This value must be between 300 (5 minutes) and 1800 (30 minutes),
+        /// inclusive.
+        /// </summary>
+        [JsonProperty("expires_in")]
+        public int ExpiresIn { get; set; }
```

Comment:
Since this is an optional property, we can use a nullable type here to use `null` to represent the absence of a value. We can then set the `NullValueHandling` to omit the property if it is `null`:

```cs
[JsonProperty("expires_in", NullValueHandling = NullValueHandling.Ignore)]
public int? ExpiresIn { get; set; }
```